src/Sindarin-Tests/SindarinDebuggerTest.class.st

```diff
@@ -362,15 +362,16 @@ SindarinDebuggerTest >> testIsExecutionFinished [
 	| scdbg |
 	scdbg := SindarinDebugger debug: [ self helperMethod16 ].
 	self deny: scdbg isExecutionFinished.
+	
 	scdbg
 		stepOver;
 		step;
 		step;
 		stepOver;
 		step;
-		stepOver;
-		step.
-	"Reached `self isActiveProcess` in Process>>#terminate"
+		stepOver.
+	"Reached `Processor terminateRealActive` method.
+	It is a special termination method which is considered as the end for simulation and should not be stepped" 
```

Comment:
Please sync this changes with actual repository if it exists. I did not find a separate project for Sindarin